### ISSUE TITLE
On Android Copy Button not show 

### ISSUE THREAD

**Issue by imamrobani** [tttstudios/react-native-otp-input#111]
on my Android device not show copy button ?
<img width="345" alt="Screen Shot 2020-12-11 at 17 09 01" src="https://user-images.githubusercontent.com/26439320/101890696-9e9a9080-3bd3-11eb-8338-dcf05dd39315.png">

**Comment by peanut-shi** [comment]
I have the same issue

**Comment by rutul089** [comment]
I have same issue

**Comment by ntt97** [comment]
i have the same issuae

**Comment by CorentinMagerotte** [comment]
I had the same issue and I have found a solution.

Your SMS must end with "code : <your_code>" like "Your code : 1515".

If your SMS is like "Your code is : 1515" it's doesn't work 🤷‍♂️

**Comment by imamrobani** [comment]
@CorentinMagerotte correct. but I must manual go to my Inbox and then copy, and then when back to my apps it's autofill
my message is " Your code is: <otp>"
